src/Utils/queryUtils.js

```diff
@@ -10,13 +20,21 @@ export function queryForID(data, id) {
 
 export function queryForCollection(data, body) {
   let query = data;
-  const { limit, filters } = body;
+  const { limit, filters, orderBy, descending } = body;
   if (filters) {
     query = applyFilters(query, filters);
   }
   if (limit) {
     query = query.limit(Number(limit));
   }
-  return query.get().then(snapshot => snapshot.docs.map(doc => ({ ...doc.data(), id: doc.id })));
+  if (orderBy) {
+    const orderByArr = typeof orderBy === "string" ? [orderBy] : orderBy;
```

Comment:
`const orderByArr = Array.isArray(orderBy) ? orderBy : [orderBy];`

Reply:
@LucasZibell Please be gently and explain why this proposal

Reply:
I think the reason is clear enough :D